### DISCUSSION TITLE
Email encryption

### DISCUSSION THREAD

**Discussion by CDNTEK** [modoboa/modoboa#3220] [Ideas]
Greetings to everyone involved in the Modoboa project development. Your dedication to this splendid endeavor is truly appreciated. Two days ago, I embarked on the journey with Modoboa and have been thoroughly impressed with my experience thus far. I come bearing two inquiries and a proposition. Firstly, is there an issue with the calendar link functionality? Secondly, does autodiscover is not working with Mac Email client? As for my suggestion, the integration of Email encryption, such as OpenPGP, would be immensely beneficial.

**Comment by tonioo** [comment]
@CDNTEK Thank you. Indeed, there might be an issue with the latest release of the calendar plugin, that's something we must check. About Mac email client, I can't answer because I'm a Thunderbird user...
And about encryption, that's something we can consider. What kind of integration would you see?

**Comment by Aegis8** [comment]
@CDNTEK Look at this issue: https://github.com/modoboa/modoboa-installer/issues/521, perhaps it will help you. We have Mac, Windows and Linux clients and autodiscover works for all of them. Also check that your DNS entries are created as required for automx.

**Comment by xinomilo** [comment]
concerning encryption, 
openpgp is usually a client thing (mailvelope browser plugin, email client feature). 
modoboa could perhaps add opengpg feature to webmail, but having private keys stored on server? not a good practice imho.  so i'd rather keep encryption -> client side.   (another option is deltachat as email client.)

and there's storage encryption eg. by using dovecot mail-crypt plugin: https://doc.dovecot.org/2.3/configuration_manual/mail_crypt_plugin/
haven't tried it, but this looks interesting..
